rotkehlchen/tests/api/test_bitcoin.py

```diff
@@ -56,28 +59,26 @@
     '1DNg5csVnUkheYczaBECsfby8ZcmZkYnJK',
 ]
 
+TEST_BITCOIN_XPUB_1 = 'xpub6DCi5iJ57ZPd5qPzvTm5hUt6X23TJdh9H4NjNsNbt7t7UuTMJfawQWsdWRFhfLwkiMkB1rQ4ZJWLB9YBnzR7kbs9N8b2PsKZgKUHQm1X4or'  # noqa: E501
+TEST_BITCOIN_XPUB_2 = 'xpub68V4ZQQ62mea7ZUKn2urQu47Bdn2Wr7SxrBxBDDwE3kjytj361YBGSKDT4WoBrE5htrSB8eAMe59NPnKrcAbiv2veN5GQUmfdjRddD1Hxrk'  # noqa: E501
+
+
+def _check_xpub_addition(rotki: Rotkehlchen, xpubs: List[str]):
```

Comment:
What are you doing here? Why are you changing the logic of the checks?

Reply:
Previously accounts and xpubs addition endpoints were returning balances updates. Now they don't (since on addition balances are not modified). So we can't check balances by checking the response and thus we check balances that are stored in the rotkehlchen instance.

Btw, the decision about not returning balances update was discussed on one of the standups. But I think it could make sense to return balances update on account removal (since balances change) or maybe we could even leave balances update return on every addition / deletion request?

Reply:
I already mentioned in the other thread how you should handle tests now. Add queries where they were needed.